openapi/components/parameters.yaml

```diff
@@ -251,6 +251,20 @@ platform:
   schema:
     type: string
   description: The platform the asset supports.
+noplatform:
+  name: noplatform
+  in: query
+  required: false
+  schema:
+    type: string
+  description: The platform the asset does not support.
```

Comment:
This description isn't really self explanation, can you add a bit more explaining it?